### PR TITLE
Bookings seeds

### PR DESCRIPTION
**Pull request by spioupiou**
Created some bookings seeds and I also added one association - I think a kondo should have many bookings
Screen Shot 2022-01-23 at 19 09 47
Screen Shot 2022-01-23 at 19 10 00

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -38,15 +39,29 @@
...
+
+10.times do
+  bookings = Booking.new(
+    user_id: [user1, user2, user3, user4].sample.id,
+    kondo_id: kondos.sample.id,
+    confirmed: false,
+    booked_date: Time.now
+  )
+  bookings.save!
+end
```
Comment (carlnoval) [inline db/seeds.rb:67]: Minor, however please do add a puts to indicate that booking creation has been completed.

Comment (spioupiou) [reply db/seeds.rb:67]: Noted! It's in my second commit (I realized right before opening the pull request so I committed twice, sorry about the confusion!

File: db/seeds.rb [hunk 2]
```diff
@@ -65,3 +65,5 @@
   )
   bookings.save!
 end
+
+puts "Bookings Creation done!"
```
Comment (spioupiou) [inline db/seeds.rb:69]: right here :D

File: db/seeds.rb (hunk 1 above)
Comment (carlnoval) [reply db/seeds.rb:67]: No worries sorry for missing it.

**Pull request merged**